config.rb

```diff
@@ -25,7 +25,26 @@
   blog.per_page = 10
 end
 
-page "/blog/feed.xml", layout: false
+# middleman-blog does not allow the `mount_at_root` like feature
+# so we define two blog instances for English one and localized one.
+
+# English blog at /blog
+activate :blog do |blog|
+  blog.name = 'en'
+  # This will add a prefix to all links, template references and source paths
+  blog.prefix = "blog"
+  configure_blog(blog)
+end
+
+# Localized blogs at /{lang}/blog
+activate :blog do |blog|
+  blog.name = 'i18n'
```

Comment:
When there're multiple blog instances in a single middleman site (i.e. when defined `activate :blog` multiple times), we need to name each of them and specify the name in some helper methods (`blog`, `page_articles`). Otherwise an error will be raised:

* https://github.com/middleman/middleman-blog/blob/8bd8dfa/lib/middleman-blog/helpers.rb#L34-L37